html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label><br />
+        <input type="email" /><br />
+      </div>
+
+      <div>
+        <label for="radio-button">Why is coding cool</label><br />
+        <select id="is-coding-cool" name="simple">
+          <br />
+          <option selected>Select an option</option>
+          <option>Becuase it&#39;s fun</option>
+          <option>Because it&#39;s interesting</option>
+          <option>Because it&#39;s powerful</option></select
+        ><br /><br />
+      </div>
+
+      <div>
+        <label for="fav-lang">Favorite Language &#40;choose one&#41;</label
+        ><br /><br />
```

Comment:
Same thing here for the `div` instead of `br` but also you won't need a `label` here because it won't direct a user anywhere with it. This should just be a `p` element since it is just going to be text content.